code/modules/events/sentience.dm

```diff
@@ -26,9 +26,43 @@
 	SA.health = SA.maxHealth
 	SA.del_on_death = FALSE
 	greet_sentient(SA)
+	var/objtype = pick(subtypesof(/datum/objective/sentFluff/))
+	var/datum/objective/sentFluff/O = new objtype()
+	SA.mind.objectives += O
+	to_chat(SA, "<B>Purpose in life</B>: [O.explanation_text]")
```

Comment:
i didn't mean it as a refrence..but now that you mention it.....IT DOES make sense though..its not an objective..it is a reason to live...DO NOT LOSE THAT PAIR OF PANTS...